third-party/linshare/src/test/java/org/apache/james/linshare/LinshareTest.java

```diff
@@ -58,4 +65,14 @@ void linshareShouldStart() {
         .then()
             .statusCode(HttpStatus.SC_OK);
     }
+
+    @Test
+    void linshareShouldShouldHaveATechnicalAccountConfigured() {
+        List<TechnicalAccountDTO> technicalAccounts = linshareExtension.getAllTechnicalAccounts(LinshareFixture.ADMIN_ACCOUNT);
+
+        assertThat(technicalAccounts).anySatisfy(account -> SoftAssertions.assertSoftly(softly -> {
+            softly.assertThat(TECHNICAL_ACCOUNT.getUsername()).isEqualTo(account.getName());
```

Comment:
The side of this assertion is bad. An assertion should be 'english readable', and follow the `assertThat(actual).isEqualTo(expected);`

Hence here we should have:

```
softly.assertThat(account.getName()).isEqualTo(TECHNICAL_ACCOUNT.getUsername());
```